### PR TITLE
caesium-image-compressor: Add version 2.1.0

### LINKED ISSUE #8793: [Request] Caesium Image Compressor
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** Caesium Image Compressor

**Description:** Caesium is an image compression software that helps you store, send and share digital pictures, supporting JPG, PNG and WebP formats.

**Homepage:** https://saerasoft.com/caesium

**Download link(s):** [[URL(s)/link(s)]](https://github.com/Lymphatus/caesium-image-compressor/releases/tag/v2.1.0)

**Some indication of popularity/repute:** **89** forks and **1.1k** stars as of 2022-07-03

### PR DESCRIPTION
**Pull request by bookyue**
Signed-off-by: Kyle Yue <sevenbookyue@gmail.com>

Closes #8793

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/caesium-image-compressor.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "description": "Caesium is an image compression software that helps you store, send and share digital pictures, supporting JPG, PNG and WebP formats. You can quickly reduce the file size (and resolution, if you want) by preserving the overall quality of the image.",
+    "version": "2.1.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Lymphatus/caesium-image-compressor/releases/download/v2.1.0/caesium-image-compressor-2.1.0-win.zip",
+            "hash": "7efc69436328870d3f8f3e736b38043c67fd4c1883d5982d3a3e0a9af8a109f1",
+            "extract_dir": "caesium-image-compressor-2.1.0-win"
+        }
+    },
+    "bin": "Caesium Image Compressor.exe",
```
Comment (issaclin32) [inline bucket/caesium-image-compressor.json:13]: Does the app support command-line parameters? If not, then `bin` is not needed.

File: bucket/caesium-image-compressor.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://saerasoft.com/caesium",
```
Comment (issaclin32) [inline bucket/caesium-image-compressor.json:2]: Please put items in the order of `version`, `description`, `homepage`, `license`. See other existing packages for example.

**Comment by bookyue** [comment]
Thanks for your correction.
To make the commit history tidy, I rebased and fixed this PR.

**Comment by issaclin32** [comment]
config location: `$Env:AppData\SaeraSoft\Caesium Image Compressor`
(This is a reminder. No need to modify anything regard to this.)

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**